lib/kms.js

```diff
@@ -0,0 +1,17 @@
+const { KMSClient, DecryptCommand } = require('@aws-sdk/client-kms')
+const logger = require('./logger')
+
+const decrypt = async (encrypted) => {
+  try {
+    const client = new KMSClient({
+      region: process.env.AWS_REGION || 'us-east-1'
+    })
+    const command = new DecryptCommand(encrypted)
```

Comment:
I think the constructor to `DecryptCommand` [takes something different](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-kms/classes/decryptcommand.html) than just the encrypted string? e.g. 
```
 new DecryptCommand({ CiphertextBlob: Buffer.from(encrypted, 'base64') })
```
You'll see errors if you run `sam invoke..`